### PR TITLE
Unit Test before statusBar and Conflict&Merge

### PR DESCRIPTION
**Pull request by dwy189**
I need to submit this pull request to test the latest code for statusBar and Projects conflict & merge in master.

### CODE REVIEW

File: lib/atomic-management.js [hunk 1]
```diff
@@ -259,6 +261,9 @@ export default {
       atom.enablePersistence || configName || atom.config.mainSource
     );
     this.updateStatusBar();
+    //for test
+    savedConfigFile = path.join(atom.project.getPaths()[0], "/spec/test.cson")
+    CSON.writeFile(savedConfigFile, atom.config.get(), () => console.log("error"))
```
Comment (tyleri) [inline lib/atomic-management.js]: Why is this necessary?

Comment (dwy189) [reply lib/atomic-management.js]: This is for testing all the configuration settings. Currently, there is no API to get the current configuration in the project. This CSON write will write all the current configuration settings to a cson file and we will compare the cson file and config.cson for any difference.

Comment (tyleri) [reply lib/atomic-management.js]: Won't this affect the user's local project directory though?

Comment (dwy189) [reply lib/atomic-management.js]: It will  create a cson file called: "test.cson" in the spec directory. Is that a bad behavior?

Comment (tyleri) [reply lib/atomic-management.js]: I think so...
@djh329 @josephch405 what do you think?
